src/lib.rs

```diff
@@ -130,6 +130,13 @@ mod init;
 pub mod pbkdf2;
 pub mod rand;
 
+mod limb;
+
+// Really a private method; only has public visibility so that C compilation
+// can see it.
+#[doc(hidden)]
+pub use limb::GFp_rand_mod;
```

Comment:
I learned that the `public-from-private` or whatever error is generated by a lint, and we might be able to turn that lint off using `#[allow(...)]`. I'll file a follow-up for trying that.